content/en/docs/reference/scheduling/config.md

```diff
@@ -43,23 +70,45 @@ extension points:
    filtering phase. The scheduler will then select the node with the highest
    weighted scores sum.
 1. `Reserve`: This is an informational extension point that notifies plugins
-   when resources have being reserved for a given Pod.
+   when resources have being reserved for a given Pod. Plugins also implement an
+   `Unreserve` call, that gets called in the case of failure during of after
+   `Reserve`.
 1. `Permit`: These plugins can prevent or delay the binding of a Pod.
 1. `PreBind`: These plugins perform any work required before a Pod is bound.
 1. `Bind`: The plugins bind a Pod to a Node. Bind plugins are called in order
    and once one has done the binding, the remaining plugins are skipped. At
    least one bind plugin is required.
 1. `PostBind`: This is an informational extension point that is called after
    a Pod has been bound.
-1. `UnReserve`: This is an informational extension point that is called if
-   a Pod is rejected after being reserved and put on hold by a `Permit` plugin.
+
```

Comment:
for PreFilter above, can you update the text to indicate that PreFilter can return an unschedulable status indicating that all nodes are filtered for the pod being scheduled.

Reply:
I think the description already is enough for users of component config. Such detail could go in https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/

Reply:
I think it is not clear that it can act as an all-nodes Filter, but this is minor.

Reply:
See new commit